Descrcibe empty topics in 20.1 (#1347)
DevExpress/devextreme-documentation: api-reference/_hidden/dxDropDownButtonItem/onClick.md

---
---
##### shortDescription
A handler for the [click]({basewidgetpath}/Events/#itemClick) event raised for a certain item in the drop-down field.

##### param(e): Object
Information about the event.

##### field(e.component): dxDropDownButton
The widget's instance.

##### field(e.element): dxElement
#include common-ref-elementparam with { element: "widget" }

##### field(e.event): event
#include common-ref-eventparam

##### field(e.jQueryEvent): jQuery.Event

> The jQuery event that caused the handler execution. Deprecated in favor of the **event** field.

Reviewer: ```suggestion
The jQuery event that caused handler execution. Deprecated in favor of the **event** field.
```

##### field(e.model): Object

> Model data. Available only if you use Knockout.

Reviewer: ```suggestion
Model data. Available only with the use of Knockout.
```
